Symfony/CS/Fixer/Contrib/OrderedUseFixer.php

```diff
@@ -35,8 +35,7 @@ class OrderedUseFixer extends AbstractFixer
     private static $sortType = self::SORT_ALPHA;
 
     /**
-     * Array of supported sort types:
-     *  alpha, length.
+     * Array of supported sort types: alpha, length.
```

Comment:
The constants are inside the array. The fact they have those values doesn't matter.